### ISSUE TITLE
Dead Link in Releases and Cov-Lineages.org page

### ISSUE THREAD

**Issue by DBecker7** [cov-lineages/pangoLEARN#30]
Both reference a supporting information file and provide the following link:

https://github.com/cov-lineages/pangoLEARN/tree/master/pangoLEARN/supporting_information

However, this link results in a 404 error.

**Comment by AngieHinrichs** [comment]
Looks like the supporting_information directory was removed in June:
```
commit a8053be38a8fd6404c7f6d90a7b254840cafec0d
Author: aineniamh <aine.otoole@ed.ac.uk>
Date:   Thu Jun 10 12:00:05 2021 +0100

    updating init

 pangoLEARN/__init__.py                                     |   6 ++++++
 pangoLEARN/supporting_information/.DS_Store                | Bin 6148 -> 0 bytes
 pangoLEARN/supporting_information/data_prep_description.md |  16 ----------------
 3 files changed, 6 insertions(+), 16 deletions(-)
```
Probably these two sentences should be removed from https://cov-lineages.org/resources/pangolin/pangolearn.html:

> We have pulled out informative sites and this information is included in the data release on pangoLEARN. The top SNPs that are most positively and negatively associated with a given lineage are detailed in those files.

**Comment by corneliusroemer** [comment]
Yup, removed here: https://github.com/cov-lineages/pangoLEARN/commit/a8053be38a8fd6404c7f6d90a7b254840cafec0d

It's remarkably hard to search Github for deleted files, couldn't find a way that doesn't use git. How did you do it @AngieHinrichs? Like this `git log --all --full-history -- <path-to-file>`?

**Comment by AngieHinrichs** [comment]
Ah, I just did `git log --stat` and then searched for supporting_information.
